frontend/pages/events/index.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { useEffect, useState } from 'react';
+import { useTranslation } from 'next-i18next';
+import { useEventsQuery, useNewsPageInfoQuery } from '~/generated/graphql';
+import ArticleSet from '~/components/News/articleSet';
+import NewsStepper from '~/components/News/newsStepper';
+import { Grid } from '@mui/material';
+import { useRouter } from 'next/router';
+import { serverSideTranslations } from 'next-i18next/serverSideTranslations';
+import DefaultLayout from '~/layouts/defaultLayout';
+import EventSet from '~/components/Calendar/EventSet';
+
+export default function EventsPage() {
+  const router = useRouter();
+  const [pageIndex, setPageIndex] = useState(0);
+  const { t } = useTranslation('common');
+
+  const { loading, data } = useEventsQuery();
+
+  useEffect(() => {
+    const pageNumberParameter = new URLSearchParams(router.asPath).get('page');
+    const pageNumber = pageNumberParameter ? parseInt(pageNumberParameter) : 0;
```

Comment:
const pageNumber = parseInt(pageNumberParameter) || 0; is better since every input that would resolve in NaN would become 0.